std/conv.d

```diff
@@ -230,7 +221,7 @@ template to(T)
  * When converting strings _to numeric types, note that the D hexadecimal and binary
  * literals are not handled. Neither the prefixes that indicate the base, nor the
  * horizontal bar used _to separate groups of digits are recognized. This also
- * applies to the suffixes that indicate the type.
+ * applies _to the suffixes that indicate the type.
```

Comment:
> we'd get tons of PRs.

That's not a problem, annotate it with [TRIVIAL, DDOC] and it'll be merged quickly.